lib/travis/api/v3/queries/jobs.rb

```diff
@@ -1,20 +1,10 @@
 module Travis::API::V3
   class Queries::Jobs < Query
-    #params :state, :event_type, :previous_state, prefix: :job
-    #params :name, prefix: :branch, method_name: :branch_name
-
-    #sortable_by :id, :started_at, :finished_at
-
     def find(build)
       sort filter(build.jobs)
     end
 
     def filter(list)
```

Comment:
it's been commented out for ages, long before this work. I've just deleted it now. - so no we don't need it